components/support/nimbus-cli/Cargo.toml

```diff
@@ -19,3 +19,5 @@ console = "0.15.5"
 glob = "0.3.1"
 heck = "0.4.1"
 whoami = "1.4.0"
+update-informer = { version = "1.0.0", default_features = false }
```

Comment:
This appears to be a new package to app-services — I think we need to add its repo to the root `Dependencies.md` file